source/LabelGeometry.ts

```diff
@@ -0,0 +1,158 @@
+
```

Comment:
Minor: filename should be lower case to be consistent with the rest of the project.

Reply:
done, thanks!